Comparing source compatibility of /home/jenkins/.m2/repository/org/apache/jspwiki/jspwiki-tika-searchprovider/2.11.2/jspwiki-tika-searchprovider-2.11.2.jar against /home/jenkins/.m2/repository/org/apache/jspwiki/jspwiki-tika-searchprovider/2.11.0/jspwiki-tika-searchprovider-2.11.0.jar
WARNING: You are using the option '--ignore-missing-classes', i.e. superclasses and interfaces that could not be found on the classpath are ignored. Hence changes caused by these superclasses and interfaces are not reflected in the output.
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.search.tika.TikaSearchProvider  (not serializable)
	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
	===  UNCHANGED SUPERCLASS: n.a.
	===  UNCHANGED CONSTRUCTOR: PUBLIC TikaSearchProvider()
	===  UNCHANGED METHOD: PROTECTED java.lang.String getAttachmentContent(org.apache.wiki.api.core.Attachment)
